databases/week2/studentdb.sql

```diff
@@ -0,0 +1,27 @@
+CREATE DATABASE schoolstudent_db;
+USE schoolstudent_db;
+-- class table content 
+CREATE TABLE class(
+id int unsigned primary key auto_increment,
+classname varchar(255) not null,
+begins date,
+ends date
+);
+-- student table content
+CREATE TABLE student(
+id int unsigned primary key auto_increment,
+studentname varchar(255) not null,
+email varchar(255) not null unique,
+phone int unique,
+class_id int unsigned not null,
```

Comment:
class_id is not belong the to students table.
It is belong the the class_students